core/controllers/feedback_test.py

```diff
@@ -462,14 +465,12 @@ def test_create_and_fetch_suggestions(self):
         self.assertEqual(len(threads), 6)
 
         # Get a suggestion.
-        full_thread_id = threads[0]['full_thread_id']
-        thread_id = feedback_services.get_thread_id_from_full_thread_id(
-            full_thread_id)
+        thread_id = threads[0]['thread_id']
         response_dict = self.get_json(
             '%s/%s/%s' % (feconf.FEEDBACK_THREAD_URL_PREFIX, self.EXP_ID,
                           thread_id))
-        self.assertEqual(
-            response_dict['suggestion']['id'], full_thread_id)
+        self.assertEqual(feedback_services.get_thread_id_from_full_thread_id(
```

Comment:
I changed the list response (the list of threads when first loading the feedback tab) to return only thread_id. However, the response to loading a particular thread includes the suggestion's id, which is the full_thread_id. This value isn't actually used anywhere, so I didn't feel the need to change it.